compiler/integration-tests/package.json

```
@@ -1,35 +1,19 @@
 {
   "name": "integration-tests",
   "version": "1.0.0",
-  "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha",
+    "build": "echo Integration Test build step",
+    "test": "yarn test:browser",
     "test:browser": "web-test-runner",
-    "test:node": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha",
     "test:integration:browser": "web-test-runner test//integration/browser/**/*.test.ts",
     "test:integration:browser:watch": "web-test-runner test/integration/browser/**/*.test.ts --watch"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "@esm-bundle/chai": "^4.3.4-fix.0",
-    "@web/dev-server-esbuild": "^0.3.6",
-    "@web/test-runner": "^0.15.3",
-    "@web/test-runner-webdriver": "^0.7.0",
-    "chai": "^4.3.7",
-    "fflate": "^0.8.0",
-    "mocha": "^10.2.0",
-    "smol-toml": "^1.1.2",
-    "ts-node": "^10.9.1",
-    "typescript": "^5.0.4"
-  },
   "dependencies": {
     "@aztec/bb.js": "^0.5.1",
-    "@noir-lang/acvm_js": "./../../.packages/acvm_js",
-    "@noir-lang/noir-source-resolver": "^1.1.4",
-    "@noir-lang/noir_wasm": "./../../.packages/noir_wasm",
-    "@noir-lang/noirc_abi": "./../../.packages/noirc_abi_wasm"
+    "@noir-lang/acvm_js": "^0.26.0",
+    "@noir-lang/noir-source-resolver": "1.1.4",
+    "@noir-lang/noir_wasm": "0.11.1",
+    "@noir-lang/noirc_abi": "0.8.0"
```

```suggestion
    "@noir-lang/acvm_js": "^0.26.0",
    "@noir-lang/noir-source-resolver": "workspace:*",
    "@noir-lang/noir_wasm": "workspace:*",
    "@noir-lang/noirc_abi": "workspace:*"
```

Would be best to pull whatever is latest in the workspace, no?

---

this reflects latest though?

---

This requires us to keep it in sync with the versions of the packages, otherwise we could go install from the registry. `workspace:*` enforces that we use whatever is in the workspace only.

---

This will not work as evidenced here https://github.com/noir-lang/noir/actions/runs/6184173181/job/16787329134?pr=2677#step:3:7

---

This is a [feature of yarn workspaces](https://yarnpkg.com/features/workspaces#cross-references) and I've used it successfully previously. This is likely failing due to us having a yarn v1 workspace which may not have had this feature.